src/main/java/org/opentripplanner/routing/algorithm/strategies/InterleavedBidirectionalHeuristic.java

```diff
@@ -296,15 +301,27 @@ public void doSomeWork() {
                         maxWeightSeen = weight;
                     }
                 }
-                continue;
+                if (!stopReached) {
+                    stopReached = true;
+                    rr.softWalkLimiting = false;
+                    rr.softPreTransitLimiting = false;
+                    if (s.walkDistance > rr.maxWalkDistance) {
+                        // Add 300 meters in order to search for nearby stops
+                        rr.maxWalkDistance = s.walkDistance + 300;
```

Comment:
Is this necessary for the change, or is it a separate change?

Reply:
It is actually part of https://github.com/entur/OpenTripPlanner/commit/79144c14582f285c7221a2842bdd28172eec20b6 from the Finnish branch. It seems to widen the search if no stops are found. It is not really necessary, so it could probably be removed from this pull request.